homeassistant/components/reolink/strings.json

```diff
@@ -8,13 +8,15 @@
           "host": "[%key:common::config_flow::data::host%]",
           "port": "[%key:common::config_flow::data::port%]",
           "use_https": "Enable HTTPS",
+          "baichuan_port": "Basic Service Port",
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         },
         "data_description": {
           "host": "The hostname or IP address of your Reolink device. For example: '192.168.1.25'.",
           "port": "The port to connect to the Reolink device. For HTTP normally: '80', for HTTPS normally '443'.",
           "use_https": "Use a HTTPS (SSL) connection to the Reolink device.",
+          "baichuan_port": "The 'Basic Service Port' to connect to the Reolink device. Normally '9000' unless manually changed in the Reolink desktop client.",
```

Comment:
I think you should rewrite the description as we have now multiple ports for connecting to the device